zed/Makefile

```diff
@@ -20,13 +20,13 @@ plugin.opt.so: build/Release/Makefile
 build/Debug/Makefile:
 	mkdir -p build/Debug && \
 	cd build/Debug && \
-	cmake -DCMAKE_BUILD_TYPE=Debug  -DCMAKE_CXX_COMPILER=$(CXX) -DCMAKE_C_COMPILER=$(CC) ../.. && \
+	cmake -DCMAKE_BUILD_TYPE=Debug  -DCMAKE_CXX_COMPILER=$(CXX) -DCMAKE_C_COMPILER=$(CC) -DCMAKE_LIBRARY_PATH=/usr/local/cuda/lib64/stubs ../.. && \
```

Comment:
The user's CUDA may not be here. Furthermore, why is this required in the first place?